File: EnBuyukDeger.cs
using System;

class Program
{
    static int max_(int[] x)
    {
        int enBuyuk = x[0];

        for (int i = 1; i < x.Length; ++i)
        {
            if (x[i] > enBuyuk)
            {
                enBuyuk = x[i];
            }
        }

        return enBuyuk;
    }

    static void Main()
    {
        int a1, a2, a3;

        Console.Write("1. sayı : ");
        a1 = Convert.ToInt32(Console.ReadLine());

        Console.Write("2. sayı : ");
        a2 = Convert.ToInt32(Console.ReadLine());

        Console.Write("3. sayı : ");
        a3 = Convert.ToInt32(Console.ReadLine());

        int[] dizi = { a1, a2, a3 };

        int sonuc = max_(dizi);

        Console.WriteLine("En büyük sayı: " + sonuc);
    }
}

File: EnBuyukDeger.py
def max_(x):
    enBüyük = x[0]

    for i in x:
        if i > enBüyük:
            enBüyük = i
    return enBüyük

s1 = int(input("1. sayı : "))
s2 = int(input("2. sayı : "))
s3 = int(input("3. sayı : "))

list = [s1,s2,s3]
print("En büyük sayı : " + str(max_(list)))